src/lib.rs

```diff
@@ -2033,4 +2179,117 @@ mod test {
         assert_eq!(products[0], products[1]);
         assert_eq!(products[0], products[2]);
     }
+
+    mod formatter {
+        #[cfg(feature = "std")]
+        #[test]
+        fn format_as_decimal() {
+            use super::{Ratio, Zero};
+
+            let data: [(u8, u8, &'static str, bool); 60] = [
```

Comment:
This table can be excluded from formatting: `#[cfg_attr(rustfmt, rustfmt_skip)]`
(The newer `#[rustfmt::skip]` won't work with older rustc.)